platform/view/services/comm/comm_test.go

```diff
@@ -42,7 +63,16 @@ func TestWebsocketSession(t *testing.T) {
 		pkID:        []byte("bob"),
 	}
 
-	testExchange(aliceNode, bobNode)
+	return aliceNode, bobNode
+}
+
+func shutdown(nodes ...Node) {
+	// TODO: how to check that the comm service is actually stopped?
```

Comment:
Stop normally waits. What is it that's failing? Maybe the connections that are not done sending?

Reply:
See bug description in https://github.com/hyperledger-labs/fabric-smart-client/issues/884